project_timesheet_time_control/tests/test_project_timesheet_time_control.py

```diff
@@ -60,15 +60,30 @@ def test_aal_from_other_employee_no_button(self):
         self.line.employee_id = self.other_employee
         self.assertFalse(self.line.show_time_control)
 
-    def test_create_write_analytic_line(self):
-        line = self.env['account.analytic.line'].create({
-            'date_time': fields.Datetime.now(),
+    def test_create_analytic_line(self):
+        line = self._create_analytic_line(datetime(2016, 3, 24, 3), tz="EST")
+        self.assertEqual(line.date, date(2016, 3, 23))
+
+    def test_create_analytic_line_with_string_datetime(self):
+        line = self._create_analytic_line('2016-03-24 03:00:00', tz="EST")
+        self.assertEqual(line.date, date(2016, 3, 23))
+
+    def test_write_analytic_line(self):
+        line = self._create_analytic_line(datetime.now())
+        line.with_context(tz="EST").date_time = '2016-03-24 03:00:00'
+        self.assertEqual(line.date, date(2016, 3, 23))
```

Comment:
@pedrobaeza Notice this test.

If an AAL is created with this timezone and this time, the AAL date field should be mirrored as being the previous day, because DB dates are always stored in UTC, and technically, in UTC it would be day 23.